hw/rdc_setup.sh

```diff
@@ -29,7 +29,8 @@ rdc_setup(){
     fi
 
     SUBDIR="bf_switchd"
-    cp $RDC_BFD/$SUBDIR/CMakeLists.txt $OS_BFD/$SUBDIR/CMakeLists.txt
+    FILES=("CMakeLists.txt" "bf_switchd.c")
```

Comment:
Might be good to add a comment here why you are using this workaround, other than that LGTM

Reply:
@fruffy I don't understand what exactly you refer to as workaround. The overall reason for the change is explained in the commit message.
I've just amended the PR to include `bf_switchd.c` in the `RDC_README`.

Reply:
Well, the intent was to provide some reason why `bf_switchd.c` needs to be copied.  But I looked at the rest of the file and the use of the commands are similar enough, so there is no explanation needed. Merging.